roles/openshift_aws/tasks/sanitize.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Abort when openshift_aws_clusterid is invalid
+  when:
+    - openshift_aws_clusterid is defined
```

Comment:
I don't think this check is required since:
`openshift_aws_clusterid` will always be defined since it is in `defaults/main.yml`.